src/raft/raft.rs

```diff
@@ -1836,21 +2003,22 @@ impl<T: Storage> Raft<T> {
     // false.
     // check_quorum_active also resets all recent_active to false.
     fn check_quorum_active(&mut self) -> bool {
-        let mut act = 0;
+        // NOTE: Here is a little different from etcd.
```

Comment:
what is the difference?

Reply:
In etcd, self is always active even if it's a learner. But I think it's always active only if it's a voter but not learner. Is it correct?

Reply:
But this function can only be called if the peer is a leader.